projects/dtrinity/index.js

```diff
@@ -0,0 +1,18 @@
+const { dLendTvl, dUSDCollateralTvl, dUSDAMOTvl, dLendBorrowed } = require('./helper');
+
+const tvl = async (api) => {
+    await Promise.all([
+        dLendTvl(api),
+        dUSDCollateralTvl(api),
+        dUSDAMOTvl(api)
```

Comment:
We cant count the minted token supply as tvl, you can check our frax implementation. I refactored this code to count the frax part of frax-dusd curve lp token in your amo.